src/main/java/org/gaul/s3proxy/S3ProxyHandler.java

```diff
@@ -2437,12 +2452,8 @@ private void sendSimpleErrorResponse(
         }
     }
 
-    public void setBlobStoreLocator(BlobStoreLocator locator) {
-        this.blobStoreLocator = locator;
-    }
-
     @SuppressWarnings("serial")
-    static final class S3Exception extends Exception {
+    public static final class S3Exception extends Exception {
```

Comment:
This seems like it should move out of `S3ProxyHandler` now that it is `public`.  I can do this in a subsequent commit or you can include it in this one.